### DISCUSSION TITLE
How to get the Pair Extraordinaire badge in github.

### DISCUSSION THREAD

**Discussion by royer0219** [community/community#126639] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I was trying to get the Pair Extraordinaire badge in github. But I didn't get it. Please tell me about it in details

**Comment by queenofcorgis** [comment]
Hi @royer0219 , 

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team
